userspace/libscap/ppm_sc_names.c

```diff
@@ -70,7 +70,7 @@ const char *scap_get_ppm_sc_name(ppm_sc_code sc)
 	ASSERT(sc >= 0 && sc < PPM_SC_MAX);
 
 	/* Lazy loading */
-	if(g_ppm_sc_names[0] && (strcmp(g_ppm_sc_names[0], "") == 0))
+	if(g_ppm_sc_names[0][0] == '\0')
```

Comment:
The warning was lamenting that g_ppm_sc_names[n] is a fixed-size array and, as such, cannot be null. The change from calling `strcmp` against an empty string literal to checking that the first character in the array is `\0` is an extra that I added thinking it is equivalent, yet simpler and more efficient.
Feel free to comment on this.